kong/db/strategies/off/init.lua

```diff
@@ -213,7 +213,7 @@ local function page_for_key(self, key, size, offset, options)
     end
 
     if item then
-      ret[ret_idx] = schema:process_auto_fields(item, "select", true, PROCESS_AUTO_FIELDS_OPTS)
```

Comment:
Unnecessary.

Reply:
If @bungle agrees, I will discard the 2nd commit.

Reply:
@outsinre the DAO calls process auto fields already. I think the idea was to remove it from strategy as it then causes double process_auto_fields.

Reply:
@bungle get it. So this PR is ready for review and merge.